boards/adafruit/macropad_rp2040/adafruit_macropad_rp2040.yaml

```
@@ -0,0 +1,23 @@
+identifier: adafruit_macropad_rp2040
+name: Adafruit MacroPad RP2040
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+ram: 264
+flash: 8192
+supported:
+  - uart
+  - gpio
+  - i2c
+  - spi
+  - pwm
+  - usb_device
+  - display
+  - qdec
+  - led_strip
```

```suggestion
  - display
  - gpio
  - i2c
  - led_strip
  - pwm
  - qdec
  - spi
  - uart
  - usb_device
```
Non-blocking: sort alphabetically.

---

In the current situation, I think it would be better to align it with rpi_pico from the viewpoint of ease of maintenance. If you want to fix it, it would be better to do it all at once with the rp2040 board.